.prow.yaml

```diff
@@ -1,10 +1,442 @@
+---
```

Comment:
Well, in this specific case the three dashes are to signal the start of the document, they are not required indeed. I put them because yamllint was raising a warning.

https://yaml.org/spec/1.2/spec.html
> YAML uses three dashes (“---”) to separate directives from document content. This also serves to signal the start of a document if no directives are present.